src/components/views/settings/index.tsx

```diff
@@ -163,7 +178,21 @@ export const Settings: React.FC<SettingsProps> = ({navigation}) => {
                   accessibilityLabel={label}
                   accessibilityRole="button"
                   accessibilityHint={hint}
-                  onPress={() => navigation.navigate(screen)}>
+                  onPress={() =>
+                    screen === 'test'
+                      ? Linking.openURL(
+                          `mailto:support@example.com?subject=App Feedback for COVID Alert NY: Version ${version}&body=\n\n\n\n
```

Comment:
`\n` newlines don't work for me on Android / Gmail, `<br>` does work however (I've not tested that this also works on iOS)